UI/MapboxNavigationUI/OSRMTextInstructions.swift

```diff
@@ -0,0 +1,303 @@
+import Foundation
+import MapboxDirections
+
+// Will automatically read localized Instructions.plist
+let OSRMTextInstructionsStrings = NSDictionary(contentsOfFile: Bundle.navigationUI.path(forResource: "Instructions", ofType: "plist")!)!
+
+extension String {
+    var sentenceCased: String {
+        return String(characters.prefix(1)).uppercased() + String(characters.dropFirst())
+    }
+}
+
+class OSRMInstructionFormatter: Formatter {
```

Comment:
As I pointed out in https://github.com/mapbox/MapboxNavigation.swift/pull/42#pullrequestreview-19953072, I think it’s very important for the Swift port of OSRMTextInstructions to live [independently](https://github.com/Project-OSRM/osrm-text-instructions.swift) of the Mapbox navigation SDK. The instructions included in this PR are also out of date compared to what’s in Transifex that could be transformed via the existing script to plist format.